### PR TITLE
[qa] cleanup runflake8 and runisort

### LINKED ISSUE #62: [qa] Cleanup isort and flake8 settings in an OpenWISP module

### PR DESCRIPTION
**Pull request by 1871vinayak**
Cleaning up runisort and flake8 settings.
GCI Task : https://codein.withgoogle.com/dashboard/task-instances/6061885926408192/

Closes #62

### CODE REVIEW

**Comment by 1871vinayak** [comment]
Successfully Cleanup isort and flake8 settings in an OpenWISP module 
And my travis passed successfully

**Comment by atb00ker** [comment]
Okay. To begin with, squash all the commits and make 1 commit as per the guidelines here: http://openwisp.io/docs/developer/contributing.html?highlight=commit#commit-message-style-guidelines

File: django_ipam/compat.py [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+                                .format((a,
+                                        b)))
+            return b.network_address <= a.network_address \
+                and b.network_address >= a.broadcast_address
         except AttributeError:
-            raise TypeError("Unable to test subnet containment"
-                            "between %s and %s".format(a, b))
+            raise TypeError('Unable to test subnet'
+                            'containmentbetween %s and %s'
+                            .format(a,
```
Comment (nemesifier) [inline django_ipam/compat.py]: this block of changes are terrible, please don't do this, if you're getting a warning from flake8, let it fail and I will tell you if we can ignore it or not

File: .travis.yml [hunk 2]
```diff
@@ -17,11 +17,11 @@ env:
 
 before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
-  - ./runisort
   - ./runcheckmigration
   - npm install -g jslint
   - jslint ./django_ipam/static/django-ipam/js/*.js
+  - flake8
+  - isort --check-only --recursive --diff .
```
Comment (nemesifier) [inline .travis.yml]: please don't change the order of the checks, maintain the same order

File: setup.cfg [hunk 3]
```diff
@@ -5,6 +5,8 @@ exclude = *.egg-info,
           ./tests/*settings*.py,
           docs/*
 max-line-length = 110
+# W605: invalid escape sequence
+ignore = W605
```
Comment (nemesifier) [inline setup.cfg]: see here: https://github.com/openwisp/openwisp-controller/blob/master/setup.cfg#L16-L20

**Review by nemesifier — CHANGES_REQUESTED** [review]

File: django_ipam/compat.py (hunk 1 above)
Comment (1871vinayak) [reply django_ipam/compat.py]: Do i need to revert all the changes??

**Comment by atb00ker** [comment]
One small thing: can you please add `Closes <your-issue-number>` in your pull request description. 
It links the issue with pull request. ( Useful to navigate )

**Comment by 1871vinayak** [comment]
> 
> 
> One small thing: can you please add `Closes <your-issue-number>` in your pull request description.
> It links the issue with pull request. ( Useful to navigate )

Done 👍

File: django_ipam/compat.py [hunk 4]
```diff
@@ -1,24 +1,30 @@
...
+
             if a._version != b._version:
-                raise TypeError("%s and %s are not of the same version".format((a, b)))
-            return (b.network_address <= a.network_address and
-                    b.network_address >= a.broadcast_address)
+                raise TypeError('%s and %s are not of the same version'
+                                .format((a,
+                                        b)))
+            return b.network_address <= a.network_address \
+                and b.network_address >= a.broadcast_address
```
Comment (nemesifier) [inline django_ipam/compat.py]: I still see a lot of unnecessary changes to this file, why are you changing these lines?

File: django_ipam/compat.py [hunk 5]
```diff
@@ -1,24 +1,30 @@
-from ipaddress import IPv4Network, IPv6Network
+﻿from ipaddress import IPv4Network, IPv6Network
 
 
 def patch_ipaddress_lib():
     """
     On python < 3.7, the ipaddress library does not have
     the method ``subnet_of``, so we add it via monkey patching
     """
+
```
Comment (nemesifier) [inline django_ipam/compat.py]: do not add unneeded blank lines

File: django_ipam/compat.py [hunk 6]
```diff
@@ -1,24 +1,30 @@
...
     """
     On python < 3.7, the ipaddress library does not have
     the method ``subnet_of``, so we add it via monkey patching
     """
+
     def _is_subnet_of(a, b):
         try:
+
             # Always false if one is v4 and the other is v6
+
```
Comment (nemesifier) [inline django_ipam/compat.py]: do not add unneeded blank lines

File: django_ipam/compat.py [hunk 7]
```diff
@@ -1,24 +1,30 @@
...
+                                        b)))
+            return b.network_address <= a.network_address \
+                and b.network_address >= a.broadcast_address
         except AttributeError:
             raise TypeError("Unable to test subnet containment"
                             "between %s and %s".format(a, b))
 
     def subnet_of(self, other):
         """ Return True if this network is a subnet of other """
+
```
Comment (nemesifier) [inline django_ipam/compat.py]: do not add unneeded blank lines

**Review by nemesifier — CHANGES_REQUESTED** [review]
there are still changed that are not right, see my comments

File: django_ipam/compat.py (hunk 4 above)
Comment (1871vinayak) [reply django_ipam/compat.py]: if i'll not change then travis will not build

Comment (1871vinayak) [reply django_ipam/compat.py]: 👍 Done travis build passed

File: django_ipam/compat.py (hunk 5 above)
Comment (1871vinayak) [reply django_ipam/compat.py]: 👍 Done

File: django_ipam/compat.py (hunk 6 above)
Comment (1871vinayak) [reply django_ipam/compat.py]: 👍 Done

File: django_ipam/compat.py (hunk 7 above)
Comment (1871vinayak) [reply django_ipam/compat.py]: 👍 Done

**Comment by 1871vinayak** [comment]
Corrected the commit 👍 
@atb00ker 
@nemesisdesign

**Review by nemesifier — APPROVED** [review]
:+1:

**Comment by 1871vinayak** [comment]
Thanks @nemesisdesign 
😃
